doc/user_manual/Makefile

```diff
@@ -34,13 +34,11 @@ clean:
 libraries.tex : $(LIB_FILES)
 	python ../../scripts/library_handler.py manual > libraries.tex
 
-# right now we have a catch-22, need to build RAVEN to build docs, but need docs to know how to build RAVEN
-#   so for now we do NOT automatically generate docs, but leave tools for developers to do so in place.
-#optimizer.tex : generated/generateOptimizerDoc.py
-#	./generate_docs.sh
+optimizer.tex : generated/generateOptimizerDoc.py
+	./generate_docs.sh
```

Comment:
Why is this change reverted with regards to the rework?

Reply:
reverted back.